CONTRIBUTING.md

```diff
@@ -61,12 +61,22 @@ The most comprehensive test suite we have is the `nu-test-support` crate. For te
   ```shell
   cargo clippy --workspace -- -D warnings -D clippy::unwrap_used -A clippy::needless_collect
   ```
+  or via the `toolkit.nu` command:
```

Comment:
After working on the toolkit, I thought that it would be a good idea to document its use! 😸

Reply:
maybe that'll be too much text for `CONTRIBUTING.md`.
no super strong preferences here, even though i like the mention of the toolkit :yum:

Reply:
I can maybe combine the shell code blocks into a single one like this:
```shell
cargo test --workspace

# or via the `toolkit.nu` command:
use toolkit.nu test
test
```

Reply:
i think it's ok as it is now, we'll change it if the other members of the team do not like it of if we find later it's not appropriate :relieved: